src/robusta/integrations/slack/sender.py

```diff
@@ -645,6 +765,99 @@ def send_finding_to_slack(
             thread_ts=thread_ts,
         )
 
+    def __send_finding_to_slack_preview(
+        self,
+        finding: Finding,
+        sink_params: SlackSinkPreviewParams,
+        platform_enabled: bool,
+        thread_ts: str = None,
+    ) -> str:
+        blocks: List[SlackBlock] = []
+        attachment_blocks: List[BaseBlock] = []
+
+        slack_channel = ChannelTransformer.template(
+            sink_params.channel_override,
+            sink_params.slack_channel,
+            self.cluster_name,
+            finding.subject.labels,
+            finding.subject.annotations,
+        )
+
+        if finding.finding_type == FindingType.AI_ANALYSIS:
+            # holmes analysis message needs special handling
+            self.send_holmes_analysis(finding, slack_channel, platform_enabled, thread_ts)
+            return ""  # [arik] Looks like the return value here is not used, needs to be removed
+
+        status: FindingStatus = (
+            FindingStatus.RESOLVED if finding.title.startswith("[RESOLVED]") else FindingStatus.FIRING
+        )
+
+        if finding.title:
+            blocks.extend(self.__create_finding_header_preview(finding, status, platform_enabled,
```

Comment:
this returns SlackBlock 
Maybe we should separate it to 2 lists - one with SlackBlocks, and one with BaseBlocks (that needs to be converted to Slack Blocks)